src/curves/secp256k1.ts

```diff
@@ -40,3 +40,8 @@ export const compressPublicKey = (publicKey: Uint8Array): Buffer => {
   const point = Point.fromHex(publicKey);
   return Buffer.from(point.toRawBytes(true));
 };
+
+export const decompressPublicKey = (publicKey: Uint8Array): Buffer => {
+  const point = Point.fromHex(publicKey);
```

Comment:
Does `Point.fromHex` do the decompression here or where is it happening? Perhaps leave a comment to explain.